pkg/multitenant/tenantcapabilities/tenantcapabilitiesauthorizer/authorizer_test.go

```diff
@@ -120,8 +120,8 @@ func (m mockReader) updateState(updates []*tenantcapabilities.Update) {
 	}
 }
 
-// GetCapabilities implements the tenantcapabilities.Reader interface.
-func (m mockReader) GetCapabilities(
+// GetCapabilitiesMap implements the tenantcapabilities.Reader interface.
+func (m mockReader) GetCapabilitiesByID(
```

Comment:
Looks like this might be leftover from an interface method name change you intended to make?